rangeseekbar/src/main/java/com/innovattic/rangeseekbar/RangeSeekBar.kt

```diff
@@ -338,9 +347,10 @@ open class RangeSeekBar : View {
 	 * @param canvas the canvas to draw on
 	 * @param position the horizontal position of the drawable's left
 	 */
-	private fun Drawable.drawAtPosition(canvas: Canvas, position: Int) {
-		val top = (height - intrinsicHeight) / 2
-		setBounds(position, top, position + intrinsicWidth, top + intrinsicHeight)
+	private fun Drawable.drawAtPosition(canvas: Canvas, position: Int, offset: Point) {
```

Comment:
`offset` could be an optional argument, with default value of `Point(0, 0)`

maybe also check the javadoc for `position`, it's not very clear from the description what it does